srv/components/misc_pkgs/kafka/start.sls

```diff
@@ -22,10 +22,18 @@ Start zoopkeper:
     - cwd: /opt/kafka
     - unless: test 1 -le $(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}' | wc -l)
 
+#TODO: find better solution to add delay
+Wait for zookeeper to start:
```

Comment:
This is not a good solution.
This should be handled with a polling loop.

To be addressed against https://github.com/Seagate/cortx-prvsnr/issues/1136